tooling/noirc_dbg/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "noirc_dbg"
+description = "Debugger for Noir"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+
+[dependencies]
+acvm = { git = "https://github.com/dmvict/noir", rev = "db36e6d66b9fadfbf27f7d2852de806095f7c77e" }
```

Comment:
Does this still need to use a fork?

Reply:
> Does this still need to use a fork?

Hello @kevaundray 

Yes, the pr still need the fork. I described the changes in the first post in the issue #3094. 
[Here is the code](https://github.com/noir-lang/noir/compare/master...dmvict:noir:alt_compiler?expand=1) that allows to compile the brillig bytecode. 
I think we should continue discussion about integration of this changes in the other issue/pr. 
Once the changes will be properly integrated into repo I'll be able to remove the fork from dependencies.